### PR TITLE
Script to auto-approve registration updates

### PR DESCRIPTION
**Pull request by jwalz**
## Purpose
Registration Updates should auto-approve (at the admin level) just like the initial Registration.
## Changes
* Script to identify and approve `UNAPPROVED` SchemaResponses with submission timestamp longer than the environment's auto-approval threshold.
* Tests
* 
## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

### CODE REVIEW

File: osf/management/commands/approve_pending_schema_responses.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+from framework.celery_tasks import app as celery_app
+
+from osf.models import SchemaResponse
+from osf.utils.workflows import ApprovalStates
+
+from website.settings import REGISTRATION_APPROVAL_TIME
```
Comment (Johnetordoff) [inline osf/management/commands/approve_pending_schema_responses.py:11]: Do we really want these to be the same? Is there a reason these are both 2 days?

File: osf/management/commands/approve_pending_schema_responses.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+def approve_pending_schema_responses(dry_run=False):
+    '''Migrate registration_responses into a SchemaResponse for historical registrations.'''
+    logger.info(
+        f'{"[DRY RUN] " if dry_run else ""}'
+        f'Auto-Approving SchemaResponses submitted longer than {THRESHOLD_HOURS} hours ago'
+    )
+    # Get all non-initial SchemaResponses that have been pending Admin Approval
+    # for longer than the environment's auto-approval threshold
+    auto_approval_threshold = timezone.now() - REGISTRATION_APPROVAL_TIME
+    print(auto_approval_threshold)
```
Comment (Johnetordoff) [inline osf/management/commands/approve_pending_schema_responses.py:28]: gotcha

File: osf/management/commands/approve_pending_schema_responses.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    pending_schema_responses = SchemaResponse.objects.filter(
+        reviews_state=ApprovalStates.UNAPPROVED.db_name,
+        submitted_timestamp__lte=auto_approval_threshold,
+        previous_response__isnull=False,
+    )
+    count = 0
+
+    for schema_response in pending_schema_responses:
```
Comment (Johnetordoff) [inline osf/management/commands/approve_pending_schema_responses.py:36]: nit: I always like [enumerate](https://docs.python.org/3/library/functions.html#enumerate) for a python count.

**Review by Johnetordoff — CHANGES_REQUESTED** [review]
one mistake, but otherwise LGTM

File: osf/management/commands/approve_pending_schema_responses.py (hunk 3 above)
Comment (jwalz) [reply osf/management/commands/approve_pending_schema_responses.py:36]: Don't actually need the index from enumerate, just want to count successful operations -- though this was a good reminder that we were assuming that all operations would succeed

**Pull request merged**